supplementary/alica_ros_turtlesim/ros2/CMakeLists.txt

```diff
@@ -80,7 +87,7 @@ install(DIRECTORY include/${PROJECT_NAME}/
         DESTINATION include/${PROJECT_NAME}
 )
 
-install(DIRECTORY launch etc
+install(DIRECTORY launch etc ../etc
```

Comment:
same

Reply:
The local etc folder contains the ros specific behaviours WaitForTrigger and WaitForMsg, while the etc folder of the parent directory contains all shared plans / behaviours like SurroundLeader.